Karl/schema_karl.h

```diff
@@ -226,10 +226,11 @@ CURRENT_STRUCT(up) {
   CURRENT_FIELD_DESCRIPTION(uptime, "The uptime of the service based on the most recent keepalive, human-readable.");
 
   CURRENT_DEFAULT_CONSTRUCTOR(up) {}
-  CURRENT_CONSTRUCTOR(up)(const std::chrono::microseconds start_time_epoch_microseconds,
-                          const std::string& last_keepalive_received,
-                          const std::chrono::microseconds last_keepalive_received_epoch_microseconds,
-                          const std::string& uptime)
+  CURRENT_CONSTRUCTOR(up)
+  (const std::chrono::microseconds start_time_epoch_microseconds,
+   const std::string& last_keepalive_received,
+   const std::chrono::microseconds last_keepalive_received_epoch_microseconds,
+   const std::string& uptime)
```

Comment:
I'm strongly for spaces in `if ()`, `for ()` etc.
Looks like we should admit that `clang-format` fucks up our macros in one or another way and we'll have to use `clang-format off` for them.